doc/onboarding/view/step2.md

```diff
@@ -0,0 +1,82 @@
+
+
+## Step 2/5 : Share anonymously data
+
+### URI
+
+`:userID/share`
```

Comment:
@gregorylegarec I thougth you read the comment bellow (that I'v juste removed not to have twice the same question); here it is : 

@gregorylegarec, @m4dz, @CPatchane : agreement is fine but you give an agreement for something; this "something" is missing. Maybe : share_agreement?